tests/usage.test.ts

```diff
@@ -1,81 +1,90 @@
+
 import Ajv from 'ajv';
 import addFormats from 'ajv-formats';
-import {Validator} from 'jsonschema';
-import {describe, expect, test} from 'vitest';
-
-import {landDrainageConsent} from '../examples/data/landDrainageConsent';
-import {lawfulDevelopmentCertificateExisting} from '../examples/data/lawfulDevelopmentCertificate/existing';
-import {lawfulDevelopmentCertificateProposed} from '../examples/data/lawfulDevelopmentCertificate/proposed';
-import {listedBuildingConsent} from '../examples/data/listedBuildingConsent';
-import {planningPermissionFullHouseholder} from '../examples/data/planningPermission/fullHouseholder';
-import {planningPermissionMajor} from '../examples/data/planningPermission/major';
-import {planningPermissionMinor} from '../examples/data/planningPermission/minor';
-import {priorApprovalBuildHomes} from '../examples/data/priorApproval/buildHomes';
-import {priorApprovalConvertCommercialToHome} from '../examples/data/priorApproval/convertCommercialToHome';
-import {priorApprovalExtendUniversity} from '../examples/data/priorApproval/extendUniversity';
-import {priorApprovalLargerExtension} from './../examples/data/priorApproval/largerExtension';
-import {priorApprovalSolarPanels} from '../examples/data/priorApproval/solarPanels';
-import generatedSchema from '../schema/schema.json';
-
-const examplesToTest = [
-  lawfulDevelopmentCertificateExisting,
-  lawfulDevelopmentCertificateProposed,
-  listedBuildingConsent,
-  planningPermissionFullHouseholder,
-  planningPermissionMajor,
-  planningPermissionMinor,
-  priorApprovalExtendUniversity,
-  priorApprovalSolarPanels,
-  priorApprovalBuildHomes,
-  priorApprovalConvertCommercialToHome,
-  priorApprovalLargerExtension,
-  landDrainageConsent,
+import {Schema, Validator} from 'jsonschema';
+import digitalPlanningApplicationSchema from '../schemas/digitalPlanningApplication.json';
+import {DigitalPlanningApplication} from '../types/schemas/digitalPlanningApplication/index';
+import {describe, test, expect} from 'vitest';
+
+/**
+ * Helper function to walk /examples directory and collect generated JSON files
+ */
+const getJSONExamples = <T>(schemaPath: string): T[] => {
+  const examples: T[] = [];
+
+  const walkDirectory = (dir: string) => {
+    const files = fs.readdirSync(dir);
+
+    for (const file of files) {
+      const filePath = path.join(dir, file);
+
+      if (fs.statSync(filePath).isDirectory()) {
+        walkDirectory(filePath);
+      } else if (path.extname(filePath) === '.json') {
+        const example = JSON.parse(fs.readFileSync(filePath, 'utf8'));
+        examples.push(example);
+      }
+    }
+  };
+
+  const examplesDir = path.join(__dirname, `../examples/${schemaPath}`);
+  walkDirectory(examplesDir);
+
+  return examples;
+};
+
+const schemas = [
+  {
+    name: 'DigitalPlanningApplication',
+    schema: digitalPlanningApplicationSchema,
+    examples: getJSONExamples<DigitalPlanningApplication>(
+      'digitalPlanningApplication'
+    ),
+  },
```

Comment:
New schemas will also need to be added here, and then all their examples will get picked up by tests automatically 👍